simulator/virtual_machine.go

```diff
@@ -2747,6 +2747,72 @@ func (vm *VirtualMachine) CreateSnapshotTask(ctx *Context, req *types.CreateSnap
 	}
 }
 
+func (vm *VirtualMachine) CreateSnapshotExTask(ctx *Context, req *types.CreateSnapshotEx_Task) soap.HasFault {
```

Comment:
Can we change the existing `CreateSnapshotTask` to call this method?
We do this for example with `RetrieveProperties` calling `RetrievePropertiesEx`:
https://github.com/vmware/govmomi/blob/49947e5bbd7b83e91cf89dc2ba80daae72dddfff/simulator/property_collector.go#L715